config/betterquesting/DefaultQuests/Quests/Tier0StoneAge-AAAAAAAAAAAAAAAAAAAAAQ==/MakingBetterTool-AAAAAAAAAAAAAAAAAAAAJg==.json

```diff
@@ -70,7 +70,7 @@
       "lockedProgress:1": 0,
       "autoClaim:1": 0,
       "isSilent:1": 0,
-      "desc:8": "With aluminium and copper, you can create a material with a higher melting temperature, \"Aluminum Brass\", which can be used to cast better tools.\n\nBut first you\u0027ll need a smeltery. See the Multiblock Goals tab for details. You can also grab some extra blocks from some villages, but they don\u0027t count towards the quest.\n\nYou can then use your fancy new bronze to make better tools with a higher mining level.\n\n§3Remember, you need to gather mining xp or use an upgrade slot to get the full mining level on your bronze pickaxe."
+      "desc:8": "With aluminium and copper, you can create a material with a higher melting temperature - Aluminum Brass - which can be used to cast better tools.\n\nFor that, you need a smeltery. See the Multiblock Goals tab for details. You can also grab some extra blocks from some villages, but they don\u0027t count towards the quest.\n\nYou can then use your fancy bronze to make better tools with a higher mining level.\n\nRemember, you need to gather mining XP or use an upgrade slot with a monster head to get the full mining potential of your bronze pickaxe."
```

Comment:
level not potential. this shouldnt be vague

Reply:
Fixed in https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/pull/13426/commits/f82dfd717d648c59b3aae7ec66a89c4d6a9aa7bf